tests/modules/eido/convert/nextflow.config

```diff
@@ -0,0 +1,7 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+    withName: 'EIDO_CONVERT' {
+        ext.args = '--st-index sample -f csv -p samples=samplesheet_validated.csv'
```

Comment:
What is `-p` standing for here? Is that meant to be the output file name? If so this _should_ be moved into the module in this case (and ${prefix}.csv), as in `validate`.

For flexibility and security (to ensure only one CSV is produced), one could alos require the format as an input channel `val(format)` which is then passed to `-f ${format}`, the suffix of `-p ${prefix}.${fornat}` and also thereotially the output glob in the output channel `path(${prefix}.${csv}), emit: output`.

Alternatively, you will need to have one output channel per converstion type output, but made `optional: true, emit: output`

Reply:
We can fix this to `csv` in the pipeline-template to make sure it 'works', but some people mayeb use the module outside of nf-core pipelines, so it's good to have it flexible

Reply:
Yes, the -p stands for paths and is supposed to provide output path (this is kind of confusing but that's not mine implementation). I will try to implement your suggestion.